src/System.Management.Automation/engine/PropertyAccessor.cs

```diff
@@ -491,7 +485,6 @@ private void RemoveValueFromFile<T>(string fileName, string key)
         }
     }
 
-#endif // CORECLR
 
     internal class RegistryAccessor : ConfigPropertyAccessor
```

Comment:
Maybe the `RegistryAccessor` should be removed now to increase the code coverage of this file. @mirichmo what do you think?